doc/packaging.md

```diff
@@ -0,0 +1,334 @@
+
+The provided `config.toml.template` is a template for configuration values that applies to the services contained in this package. It may also be used as a starting point for the `config.toml` from which configuration is applied to the individual services.
+</td>
+</tr>
+<tr>
+<td>API socket files</td>
+<td>
+
+```
+/var/run/
+└── iotedge/
+    ├── mgmt.sock
+    └── workload.sock
+```
+</td>
+<td>
+
+```
+/run/
+└── iotedge/
+    ├── mgmt.sock
+    └── workload.sock
+```
+</td>
+<td>
+
+```
+/run/aziot/
+├── certd.sock
+├── identityd.sock
+├── keyd.sock
+└── tpmd.sock
+```
+</td>
+</tr>
+<tr>
+<td>Systemd service and socket files</td>
+<td>
+
+```
+/lib/systemd/system/
+├── iotedge.service
+├── iotedge.socket
+└── iotedge.mgmt.socket
+```
+</td>
+<td>
+
+```
+/lib/systemd/system/
+├── aziot-edged.service
+├── aziot-edged.mgmt.socket
+└── aziot-edged.workload.socket
+```
+</td>
+<td>
+
+```
+/lib/systemd/system/
+├── aziot-certd.service
+├── aziot-certd.socket
+├── aziot-identityd.service
+├── aziot-identityd.socket
+├── aziot-keyd.service
+├── aziot-keyd.socket
+├── aziot-tpmd.service
+└── aziot-tpmd.socket
+```
+</td>
+</tr>
+<tr>
+<td>Unix groups (used to ACL the service sockets)</td>
+<td>
+
+`iotedge` - The `iotedge.mgmt.sock` socket
+
+</td>
+<td>
+
+`iotedge` - The MR management socket
+</td>
+<td>
+
+- `aziotcs` - The CS socket
+- `aziotid` - The IS socket
+- `aziotks` - The KS socket
+- `aziottpm` - The TPM socket
+</td>
+</tr>
+<tr>
+<td>Home directories</td>
+<td>
+
+```
+/var/lib/iotedge/
+└── hsm/
+    ├── certs/
+    ├── cert_keys/
+    └── enc_keys/
+```
+</td>
+<td>
+
+```
+/var/lib/aziot/
+└── edged/
+```
+</td>
+<td>
+
+```
+/var/lib/aziot/
+├── certd/
+│   └── certs/
+├── identityd/
+└── keyd/
+    └── keys/
+```
+</td>
+</tr>
+<tr>
+<td>Package dependencies</td>
+<td>
+
+- `moby-engine`
+- `openssl`
+</td>
+<td>
+
+- `aziot-identity-service`
+- `moby-engine`
+- `openssl`
+</td>
+<td>
+
+- `openssl`
+</td>
+</tr>
+</tbody>
+</table>
+
+## Installation
+
+For full documentation refer to the official documentation to [Install or uninstall Azure IoT Edge for Linux](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-install-iot-edge?view=iotedge-2020-11). What follows is a brief summary with accompanying notes.
+
+```sh
+apt install aziot-edge
+
+sudo cp /etc/aziot/config.toml.edge.template /etc/aziot/config.toml
+
+sudo nano -w /etc/aziot/config.toml
+
+iotedge config apply
+```
+
+After installing the `aziot-edge` package, copy the `/etc/aziot/config.toml.edge.template` to `/etc/aziot/config.toml` and edit the file to provide provisioning information.  Then run `iotedge config apply` to apply the configuration to the services. This performs initialization for both the IS+KS+CS+TPMS components installed by the `aziot-identity-service` package and the MR component installed by the `aziot-edge` package.
+
+## Updating from `iotedge` to `aziot-edge`
+
+```sh
+apt remove iotedge libiothsm-std
```

Comment:
`apt remove` isn't necessary because the packages conflict, so we don't need them to do this. Then you also won't need the warning below about not `purge`ing the package.

Reply:
Removed the remove and purged the purge